src-built-in/components/windowTitleBar/src/windowTitleBarComponent.jsx

```diff
@@ -187,7 +188,14 @@ class WindowTitleBar extends React.Component {
 	onDropHandler() {
 		FSBL.Clients.WindowClient.cancelTilingOrTabbing({});
 	}
-
+	/**
+	 * Called when user double clicks on drag handle
+	 */
+	onDoubleClick() {
+		// Looks like Actions.clickMaximize checks the window state
+		// and toggles between maximize and minimize.
```

Comment:
What do you mean by "Looks like"? Is that how it works or not?

Reply:
Alright, removed `Looks like`